mlflow/keras.py

```diff
@@ -53,7 +60,7 @@ def get_default_conda_env(include_cloudpickle=False):
 
 
 def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_objects=None,
-               **kwargs):
+               keras_module=None, **kwargs):
```

Comment:
What about `log_model()`? Let's add an additional test for `log_model()` with `tf.keras` models, since it's pretty easy to forget about it.

Reply:
Updated log_model and added a test.